lib/scripts/get_view_and_download_stats_for_collection.rb

```diff
@@ -0,0 +1,15 @@
+# script for retreiving all page views and download stats from google analytics
+# To execute, from the root of this application run:
+# current_sufia/ $ bundle exec rails runner lib/scripts/get_view_and_download_stats_for_collection.rb
+
+require './collection_members_view_and_download_stats'
+
+# NUCATS Grants Repository
+nucats_grants_repository_stats = CollectionMembersViewAndDownloadStats.new(
+  "f2bf6e1d-0e32-4ce2-a52e-bb0522d5708d"
+)
+nucats_grants_repository_stats.get_stats_and_add_to_csv(type: "pageviews")
+nucats_grants_repository_stats.get_stats_and_add_to_csv(type: "downloads")
+
+nucats_grants_repository_stats.pageviews_csv_file.close
+nucats_grants_repository_stats.downloads_csv_file.close
```

Comment:
I did have it there previously, but because we call `get_stats_and_add_add_to_csv` again for child collections, the csv file could close prematurely.

Reply:
Right, we would have the file open in `get_stats_and_add_add_to_csv` too. It's the kind of thing that requires one to be careful (which the code has been), but could be done so that one doesn't have to be as wary in the first place.